### ISSUE TITLE
How does one create file indicator using this SDK?

### ISSUE THREAD

**Issue by fhightower** [ThreatConnect-Inc/threatconnect-javascript#7]
I have tried to create file indicators using variations of the following code based on the example [here](https://github.com/ThreatConnect-Inc/threatconnect-javascript/blob/master/examples/indicator_commit.html):

```javascript
indicators.owner('Example')
    .indicator('88888888888888888888888888888888')
    .type(TYPE.FILE)
    .rating(3)
    .confidence(55)
    .done(function(response) {
        console.log('response', response);
        $('#response-content').append(JSON.stringify(response, null, 4));
    })
    .error(function(response) {
        console.error('error response', response);
    })
    .commit();
```

This gives an error when run. The body of the POST request is: 
`{"":"88888888888888888888888888888888","rating":3,"confidence":55}`

Notice that there is no name for the key before the file hash. This is ultimately caused because the `postField` for file indicators is not specified [here](https://github.com/ThreatConnect-Inc/threatconnect-javascript/blob/master/threatconnect.js#L50) (the postField is added to the query [here](https://github.com/ThreatConnect-Inc/threatconnect-javascript/blob/master/threatconnect.js#L1368)). I tried adding "file" as the `postField`, but that didn't work.

I have also tried committing an object with the type of hash as exemplified below:

```javascript
indicators.owner('Example')
    .indicator({md5: '88888888888888888888888888888888'})
    .type(TYPE.FILE)
    ...
    .commit();
```

This ^ approach does not work either. Does anyone have some working examples of how one can create file indicators using this SDK?

Thanks in advanced!

**Comment by fhightower** [comment]
Ping @bcsummers : Do you know how to create a file indicator with the Javascript SDK?

**Comment by fhightower** [comment]
This has been fixed with #13 . There is an example in the indicator_commit.html file in the example directory: https://github.com/ThreatConnect-Inc/threatconnect-javascript/blob/master/examples/indicator_commit.html#L53.
